packages/ember-mirage/package.json

```diff
@@ -28,8 +28,9 @@
     "prepack": "rollup --config"
   },
   "dependencies": {
-    "@embroider/addon-shim": "^1.0.0",
-    "@embroider/macros": "^1.12.0",
+    "@embroider/addon-shim": "^1.8.6",
+    "@embroider/macros": "^1.13.1",
+    "ember-inflector": "^4.0.2",
```

Comment:
@cah-brian-gantzler this addresses the error you saw

Reply:
thanks, was just looking into creating a PR for that.